### PR TITLE
Change mistake

### PR DESCRIPTION
**Pull request by Evgenij-Kovpak**

### CODE REVIEW

File: Elevel/Application/Features/ApplicationUserFeatures/GetApplicationUserByIdQuery.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
 
                 if(user == null)
                 {
-                    throw new NotFoundException(nameof(ApplicationUser));
+                    throw new NotFoundException(nameof(Application));
```
Comment (parys) [inline Elevel/Application/Features/ApplicationUserFeatures/GetApplicationUserByIdQuery.cs:43]: what is application?

File: Elevel/Application/Features/TopicCommands/GetTopicListQuery.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var topic = _context.Topics.AsNoTracking()
+                    .Include(x => x.Id);
```
Comment (parys) [inline Elevel/Application/Features/TopicCommands/GetTopicListQuery.cs:38]: you should not include it, include is using just for collections, not for flat properties

File: Elevel/Elevel.Api/Controllers/TopicController.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Elevel.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class TopicController : BaseApiController
```
Comment (parys) [inline Elevel/Elevel.Api/Controllers/TopicController.cs:12]: please add Authorize attribute

File: Elevel/Elevel.Api/Controllers/TopicController.cs [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    [Authorize, Route("api/[controller]")]
+    public class TopicController : BaseApiController
+    {
+        public TopicController()
+        {
+
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Create( CreateTopicCommand.Request request)
```
Comment (parys) [inline Elevel/Elevel.Api/Controllers/TopicController.cs:20]: where is [FromBody] ?

File: Elevel/Elevel.Api/Controllers/TopicController.cs [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        }
+
+        [HttpGet("id:Guid")]
+        public async Task<IActionResult> GetById( GetTopicByIdQuery.Request request)
+        {
+            return Ok(await Mediator.Send(request));
+        }
+
+        [HttpDelete("{id:Guid}")]
+        public async Task<IActionResult> DeleteAsync(Guid id, DeleteTopicCommand.Request request)
```
Comment (parys) [inline Elevel/Elevel.Api/Controllers/TopicController.cs:38]: where is [FromBody] of fromRoute or FromQuery + for delete you need just Id

File: Elevel/Elevel.Api/Controllers/TopicController.cs [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        {
+            if(id != request.Id)
+            {
+                return BadRequest();
+            }
+            return Ok(await Mediator.Send(request));
+        }
+
+        [HttpPut("{id:Guid}")]
+        public async Task<IActionResult> Update(Guid id, UpdateTopicCommand.Request request)
```
Comment (parys) [inline Elevel/Elevel.Api/Controllers/TopicController.cs:48]: where is [FromBody] of fromRoute or FromQuery

**Review by parys — APPROVED** [review]

**Review by 4500zenja1 — APPROVED** [review]

**Review by ristomin — APPROVED** [review]

**Pull request merged**